airbyte-integrations/connectors/destination-astra/unit_tests/destination_test.py

```diff
@@ -24,7 +24,7 @@ def setUp(self):
             },
         }
         self.config_model = ConfigModel.parse_obj(self.config)
-        self.logger = AirbyteLogger()
+        self.logger = logging.getLogger("airbyte")
```

Comment:
the `AirbyteLogger` relies on print instead of the logging library. The main issue I'm aware of is it doesn't respect the severity, but it also means they can't be easily configured.

On the second question: yes, that shouldn't be too much of a problem. Today all loggers must be named `airbyte` because [the name is hardcoded in a config](https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/logger.py#L13-L29), but it can be parameterized.